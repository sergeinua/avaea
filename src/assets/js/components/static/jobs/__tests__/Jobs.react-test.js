import React from 'react';
import StaticJobs from '../Jobs';
import renderer from 'react-test-renderer';


test('StaticJobs component test', () => {
  var mockUser = {email: "test@gmail.com", id: 58};
  var component = renderer.create(
    <StaticJobs user={mockUser} />
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();

});
