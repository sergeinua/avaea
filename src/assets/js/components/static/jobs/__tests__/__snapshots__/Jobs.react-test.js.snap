exports[`test StaticJobs component test 1`] = `
<div>
  <div
    className="content static jobs">
    <h1
      className="skeleton">
      Jobs Content
    </h1>
  </div>
</div>
`;
